apps/condo/pages/settings/integration/[id]/index.tsx

```diff
@@ -140,7 +162,7 @@ const BillingIntegrationDetailsPage = () => {
                                     />
                                     <PageContent>
                                         <Col span={20}>
-                                            <Row gutter={[0, 50]}>
+                                            <Row gutter={[0, 40]}>
```

Comment:
it's better to move such unchanging values (literal array/object declarations) to constants, to persist variable link between renders in order not to cause unnecessary re-renders

Reply:
fixed